neusomatic/python/filter_candidates.py

```diff
@@ -259,25 +261,57 @@ def filter_candidates(candidate_record):
                 final_records.append([chrom, pos - 1, ref, alt, line])
         final_records = sorted(final_records, key=lambda x: x[0:2])
         if dbsnp:
-            filtered_bed = pybedtools.BedTool(map(lambda x:
-                                                  pybedtools.Interval(x[1][0], int(x[1][1]),
-                                                                      int(x[1][
-                                                                          1]) + 1,
-                                                                      x[1][2], x[1][3], str(x[0])),
-                                                  enumerate(final_records))).sort()
-            dbsnp = pybedtools.BedTool(dbsnp).each(lambda x:
-                                                   pybedtools.Interval(x[0], int(x[1]),
-                                                                       int(x[
-                                                                           1]) + 1,
-                                                                       x[3], x[4])).sort()
-            non_in_dbsnp_1 = filtered_bed.window(dbsnp, w=0, v=True)
-            non_in_dbsnp_2 = filtered_bed.window(dbsnp, w=0).filter(
-                lambda x: x[1] != x[7] or x[3] != x[9] or x[4] != x[10]).sort()
+            filtered_bed = tempfile.NamedTemporaryFile(
```

Comment:
@msahraeian will the temp file be in a directory which the user can specify?

Reply:
@marghoob It is a subdirectory inside the output folder. And will be deleted upon running completion.

Reply:
Thanks @msahraeian